db/migrate/20150312215548_create_videos.rb

```diff
@@ -0,0 +1,13 @@
+class CreateVideos < ActiveRecord::Migration
+  def change
+    create_table :videos do |t|
+      t.string :video_id
```

Comment:
This isn't necessary. We'll be using the YouTube link as a unique identifier, so we won't need an arbitrary extra one.